docs/book/src/reference/envtest.md

```diff
@@ -77,7 +76,17 @@ Logs from the test runs are prefixed with `test-env`.
 
 Controller-runtime’s [envtest][envtest] framework requires `kubectl`, `kube-apiserver`, and `etcd` binaries be present locally to simulate the API portions of a real cluster.
 
-For projects built with plugin v3+ (see your PROJECT file's `layout` key), the `make test` command will install these binaries to the `testbin/` directory and use them when running tests that use `envtest`.
+The `make test` command will install these binaries to the `bin/` directory and use them when running tests that use `envtest`.
```

Comment:
That is great, just one nit. I think we need to update this doc to order the info better. Note that before it was required to manually get the binaries always which is no longer the case.

So, wdyt:
a) we let the users know that they can run make test and it will call the target make setup-envtest that will install the required binaries under the bin directory by default
b) after we explain the default behaviour we can add a section like "how to customize the location where the envtest binaries will be installed"
c) then, inside of this subsection we describe how to change the arg and also we describe how it needs/can be set up in the test code.

WDYT?